ui/snyk-report-tab.html

```diff
@@ -28,7 +28,8 @@
         overflow: auto;
       }
       .iframeContainer {
-        overflow: scroll;
+        overflow: auto;
+        min-height: 100%;
```

Comment:
question: why this?

Reply:
this is required for a single issue found use-case on snyk code test in which the results html iframe is shortened vertically because it is insufficient to fill entirely.